sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py

```diff
@@ -0,0 +1,218 @@
+        :paramtype polling: bool or ~azure.core.polling.PollingMethod
+        :keyword int polling_interval: Default waiting time between two polls
+            for LRO operations if no Retry-After header is present.
+        :rtype: ~azure.core.polling.LROPoller[~azure.communication.phonenumbers.models.PhoneNumberSearchResult]
+        """
+        search_request = PhoneNumberSearchRequest(
+            phone_number_type=phone_number_type,
+            assignment_type=assignment_type,
+            capabilities=capabilities,
+            quantity=kwargs.pop('quantity', None),
+            area_code=kwargs.pop('area_code', None)
+        )
+        return self._phone_number_client.phone_numbers.begin_search_available_phone_numbers(
+            country_code,
+            search_request,
+            **kwargs
+        )
+    @distributed_trace
+    def begin_update_phone_number_capabilities(
+            self,
+            phone_number, # type: str
+            sms=None, #type: str or PhoneNumberCapabilityType
+            calling=None, #type: str or PhoneNumberCapabilityType
+            **kwargs # type: Any
+    ):
+        # type: (...) -> LROPoller["_models.AcquiredPhoneNumber"]
+        """Updates the capabilities of a phone number.
+
+        :param phone_number: The phone number id in E.164 format. The leading plus can be either + or
+            encoded as %2B, e.g. +11234567890.
+        :type phone_number: str
+        :param calling: Capability value for calling.
+        :type calling: str or ~azure.communication.phonenumbers.models.PhoneNumberCapabilityType
+        :param sms: Capability value for SMS.
+        :type sms: str or ~azure.communication.phonenumbers.models.PhoneNumberCapabilityType
+        :keyword str continuation_token: A continuation token to restart a poller from a saved state.
+        :keyword polling: Pass in True if you'd like the LROBasePolling polling method,
+            False for no polling, or your own initialized polling object for a personal polling strategy.
+        :paramtype polling: bool or ~azure.core.polling.PollingMethod
+        :keyword int polling_interval: Default waiting time between two polls
+            for LRO operations if no Retry-After header is present.
+        :rtype: ~azure.core.polling.LROPoller[AcquiredPhoneNumber]
+        """
+        return self._phone_number_client.phone_numbers.begin_update_capabilities(
+            phone_number,
+            calling=calling,
+            sms=sms,
+            **kwargs
+        )
+
+    @distributed_trace
+    def get_phone_number(
+            self,
+            phone_number, # type: str
+            **kwargs # type: Any
+    ):
+        # type: (...) -> AcquiredPhoneNumber
+        """Gets the details of the given acquired phone number.
+
+        :param phone_number: The acquired phone number whose details are to be fetched in E.164 format,
+         e.g. +11234567890.
```

Comment:
i think this is ok, to be safe using something like 555 somewhere in the number should be ok because it's a not in use area code